src/nearest_neighbors/simulations/mcar.py

```diff
@@ -139,3 +139,54 @@ def expit(x: np.ndarray) -> np.ndarray:
     # Data[Masking == 0] = Y0[Masking == 0]
     Data: np.ndarray = np.array(Y)
     return Data, Theta, Masking
+
```

Comment:
Shouldn't this entire file be deleted? Why are we adding code here? Ideally all mcar and mnar code sits inside the same class, with different parameters that may affect how we wish to do MCAR/MNAR